pillar/devstack.sls

```diff
@@ -300,3 +309,4 @@ edx:
     FORUM_USE_TCP: True
 
     edx_platform_repo: 'https://github.com/mitodl/edx-platform.git'
+    edx_platform_version: {{ edx_platform_branch }}
```

Comment:
For the edX codebase we should use our branch that is deployed to production (`mitx/ficus-1`)